README.md

```diff
@@ -1,23 +1,33 @@
 ## Enhanced Block Entities
 
-EBE is a **100% client side** mod for Minecraft on the **[Fabric](https://fabricmc.net/use/)** mod loader which aims to increase the performance of block entity rendering, as well as offer customizability via resource packs. <br/><br/>
-
-**How does it work?** EBE Makes chests (and soon to be more block entities) use baked block models rather than laggy entity models. <br/><br/>
+EBE is a **100% client side** Minecraft mod for the **[Fabric](https://fabricmc.net/use/)** mod loader which aims to increase the performance of block entity rendering, as well as offer customizability via resource packs. <br/><br/>
+**How does it work?** EBE Makes some block entities use baked block models rather than laggy entity models. <br/><br/>
+**Is it just an optimisation mod?** No, it also allows you to customise more block entities with resource packs and makes their lighting smoother. This mod also allows for toggling christmas chests among other nice improvements.<br/><br/>
 **What about animations?** The best part about EBE is that you still get to keep animations, while gaining the performance boost of baked models! Most animated block entity models will only render when absolutely necessary. <br/><br/>
-**Can I use it with Sodium?** Yes you can, but it requires some setup. I have a [guide](https://github.com/FoundationGames/EnhancedBlockEntities/discussions/7) that you can follow to get this mod performing as optimally as possible with Sodium. <br/><br/>
-
+**Can I use it with Sodium?** Yes you can, but it requires you to install the [Indium mod](https://modrinth.com/mod/indium) as well. <br/><br/>
+**What block entities does it support?**
+- Chests
+- Ender Chests
+- Signs
+- Bells
+- Beds
+<br/><br/>
 
 Requires [Fabric API](https://modrinth.com/mod/fabric-api) <br/><br/>
 
 ## FAQ and Help
+
+**Q: How do I download the mod?**<br/>
```

Comment:
I think a section for download is better than making it a FAQ question

Reply:
Done